crates/ol/chain-types/src/block.rs

```diff
@@ -165,6 +165,13 @@ impl OLBlockBody {
     pub fn l1_update(&self) -> Option<&OLL1Update> {
         self.l1_update.as_ref()
     }
+
+    /// Computes the hash commitment of this block body.
+    /// TODO: This will use SSZ merkle hashing when ready.
```

Comment:
This should be a comment not a docstring
```suggestion
    // TODO: This will use SSZ merkle hashing when ready.
```